examples/basic/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.56.1"
+      version = ">= 1.56.1, < 2.0.0"
```

Comment:
basic example should lock into lowest supported version, please change this back